change/@fluentui-react-charting-4903549d-4b4c-4971-bfce-c488ad145a47.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31291&commitId=3608156aa0553010661b60fe98e0a38b2030d6b4&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-VerticalBarChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.default.chromium.png|474| Changed|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis - Dark Mode.default.chromium.png|495| Changed|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis - RTL.default.chromium.png|474| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>